test/e2e/serial/tests/configuration.go

```diff
@@ -693,9 +697,215 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				return kcCmNamesCur
 			}).WithContext(ctx).WithTimeout(5 * time.Minute).WithPolling(10 * time.Second).Should(Equal(kcCmNamesPre))
 		})
+
+		It("[reboot_required][slow][unsched][schedrst] should be able to correctly identify topology manager policy without scheduler restarting", Label("reboot_required", "slow", "unsched", "schedrst"), func(ctx context.Context) {
+			By("getting the number of cpus that is required for a numa zone to make a TAE deployment")
+			const (
+				NUMAZonesRequired     = 2
+				hostsRequired         = 1
+				cpuResourcePercentage = 1.5 // 1.5 meaning 150 percent
+			)
+			var nrtCandidates []nrtv1alpha2.NodeResourceTopology
+			By(fmt.Sprintf("filtering available nodes with at least %d NUMA zones", NUMAZonesRequired))
+			nrtCandidates = e2enrt.FilterZoneCountEqual(nrtList.Items, NUMAZonesRequired)
+			Expect(len(nrtCandidates)).ToNot(BeNumerically("<", hostsRequired))
```

Comment:
let's make the assertion here a Skip instead of failing. It is okay that the environment is not relevant for a test, we'ld just skip instead of failing, for example:
https://github.com/openshift-kni/numaresources-operator/blob/main/test/e2e/serial/tests/resource_accounting.go#L111

Reply:
yes you're right